README.md

```diff
@@ -5,27 +5,27 @@ u-root
 
 # Description
 
-u-root is a "universal root". It's a root file system with mostly Go source with the exception of 5 binaries.
```

Comment:
Should still probably mention that "u-root" stands for "universal root" somewhere in the readme

Reply:
yeah. There is a capability in here I hate to lose sight of: the ability to build a root file system that can boot on several different architectures with the only variance being in the 5 binaries. This is doable -- we did something like it at los alamos -- but we've never had time to achieve the dream ;-)